plugins/serializers/template/template.go

```diff
@@ -0,0 +1,82 @@
+package template
+
+import (
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+type Serializer struct {
+	Template string          `toml:"template"`
+	Log      telegraf.Logger `toml:"-"`
+
+	outTemplate *template.Template
+}
+
+func (s *Serializer) Init() error {
+	// Setting defaults
+	var err error
+
+	s.outTemplate, err = template.New("template").Parse(s.Template)
+	if err != nil {
+		return fmt.Errorf("creating template failed: %w", err)
+	}
+	return nil
+}
+
+func (s *Serializer) Serialize(metric telegraf.Metric) ([]byte, error) {
+	m, ok := metric.(telegraf.TemplateMetric)
+	if !ok {
+		s.Log.Errorf("metric of type %T is not a template metric", metric)
+		return nil, nil
+	}
+	var b strings.Builder
+	if err := s.outTemplate.Execute(&b, &m); err != nil {
+		s.Log.Errorf("failed to execute template: %v", err)
+		return nil, nil
+	}
+
+	return []byte(b.String()), nil
+}
+
+func (s *Serializer) SerializeBatch(metrics []telegraf.Metric) ([]byte, error) {
+	if len(metrics) < 1 {
+		return nil, nil
+	}
```

Comment:
This isn't really required and might be harmful e.g. if someone just outputs a fixed string on serialization as in this case you will omit that fixed string. I think you should just remove this...

Reply:
Good point. I cut and pasted it from a different plugin, but you are absolutely right.

Reply:
Done.